example/documents/document-1.json

```
@@ -1,6 +1,7 @@
 {
   "openResourceDiscovery": "1.6",
```

```suggestion
  "openResourceDiscovery": "1.12",
```
Technically v1.6 doesn't support perspectives ;)

---

Done